Dockerfile

```diff
@@ -17,7 +17,10 @@ RUN yum install -y \
 		make \
 		yarn \
 		postgresql \
-		libpq-devel
+		postgresql-devel
+
+ENV PATH "${PATH}:/usr/pgsql-10/bin"                                                                 
+ENV PG_CONFIG_DIR /usr/pgsql-10/bin  
```

Comment:
Why is this required? That directory doesn't exist in the container as far as I can see. Can you add a comment?

Reply:
It exists once you install postgresql-devel. This is required for pscopg2 install to succeed

Reply:
Strange. In my container it isnt'.

```
[root@71ec33266e71 /]# yum list installed|grep -i postgres
postgresql.x86_64                       9.2.24-4.el7_8               @base      
postgresql-devel.x86_64                 9.2.24-4.el7_8               @base      
postgresql-libs.x86_64                  9.2.24-4.el7_8               @base      
[root@71ec33266e71 /]# rpm -ql postgresql|grep -i pgsql-10
[root@71ec33266e71 /]# rpm -ql postgresql-devel|grep -i pgsql-10
[root@71ec33266e71 /]# rpm -ql postgresql-libs|grep -i pgsql-10
[root@71ec33266e71 /]#
```

Reply:
hmmm, but this made the build work. Maybe I should try without the env then.

https://jenkins.inmanta.com/job/integration-tests/job/inmanta-core-docker/default/